IBX-3077: Corrected path to graphQL schema, added information to generate ibexa schema first (#1695)
ezsystems/developer-documentation: docs/api/graphql_customization.md

You can use it if your application requires custom GraphQL resources, for instance for Doctrine entities.

To do so, create a `config/graphql/types/Query.types.yaml` file. It will be used as the GraphQL query root.

In that file, add new fields that use any custom type or custom logic you require, based
on [overblog/GraphQLBundle](https://github.com/overblog/GraphQLBundle).

> The custom schema should be created only after generating other schemas to avoid problems, especially if the custom schema depends on other schema elements. For example ...

Reviewer: ```suggestion
The custom schema should be created only after generating other schemas to avoid problems, especially if the custom schema depends on other schema elements. For example:
```

> `Type "Domain" inherited by "Query" not found.`. To prevent this problem from occurring during deployment, add the generated schemas to the repository,

Reviewer: ```suggestion
`Type "Domain" inherited by "Query" not found`.
To avoid this problem during deployment, add the generated schemas to the repository.
```

> you should also remember to update the schema in the event of any changes related to graphQL as well as when changing the environment, for example from dev to prod

Reviewer: ```suggestion
Update the schema in the event of any changes related to GraphQL as well as when changing the environment, for example from `dev` to `prod`.
```
